erpnext_com/www/docs/user/manual/en/selling/sales-partner.md

```diff
@@ -38,6 +38,16 @@ https://yourCompanyName.erpnext.com/partners
 
 <img class="screenshot" alt="Sales Partner" src="{{docs_base_url}}/assets/img/crm/sales-partner-listing.png">
 
+### 2.4 Track Sales via Sales Partner
+
+Sales Partner can actively generates leads for your company products/ services. To track the performance of each sales partner use Referral Code and their URL as below
+
+URL as shown below should be sent to the sales partner to use in their website/campaign.
+
+e.g. A URL having "sp" as paramter like this http://xyz.erpnext.com?sp=speed will capture the Sales Partner Informatiton in the Sales Order generated via Shopping Cart.
+
+<img class="screenshot" alt="Sales Partner Referral Code" src="{{docs_base_url}}/assets/img/selling/sales-partner-refrral-code.png">
```

Comment:
In all the screenshots, please use a red box to highlight the field

Reply:
red box done for all images, will update and let you know, meanwhile can you point me to any standard screenshot , so that i can resize images accordingly.

Reply:
Just ensure your screenshots are clean and there's no cut off text, padding etc is not unequal

Reply:
screenshots improved.